$NetBSD: patch-configure,v 1.2 2019/12/12 00:13:52 nia Exp $

We can't predict the uid/gid the user will be assigned before the package
is installed.

--- configure.orig	2019-10-25 10:40:44.000000000 +0000
+++ configure
@@ -224,10 +224,6 @@ This is almost never what you should do.
 unprivileged user/group to build and run as or pass the '--gid [id|name]' flag
 to specify an unprivileged group to run as.
 EOW
-		if (!prompt_bool $interactive, "Are you sure you want to build as the $group[0] group?", 0) {
-			say STDERR "If you are sure you want to build as the $group[0] group pass the --gid $group[2] flag." unless $interactive;
-			exit 1;
-		}
 	}
 }
 $config{GROUP} = $group[0];
@@ -250,10 +246,6 @@ This is almost never what you should do.
 unprivileged user/group to build and run as or pass the '--uid [id|name]' flag
 to specify an unprivileged user to run as.
 EOW
-		if (!prompt_bool $interactive, "Are you sure you want to build as the $user[0] user?", 0) {
-			say STDERR "If you are sure you want to build as the $user[0] user pass the --uid $user[2] flag." unless $interactive;
-			exit 1;
-		}
 	}
 }
 $config{USER} = $user[0];
